sapp/sarif.py

```diff
@@ -42,6 +55,7 @@ class SARIF:
     _tool_warning_code_ranges: Dict[str, Tuple[int, int]] = field(default_factory=dict)
```

Comment:
I don't this should be a `dataclass` anymore /cc @arthaud  wdyt?

Reply:
right, a `dataclass` with a "private" mutable member is odd. Maybe we could just make this a normal class.
Since it's pre-existing I guess it's fine to keep or fix in a follow-up.